main.c

```diff
@@ -134,7 +138,8 @@ main(int argc, char *argv[])
 
 	/* Prompt for a password. */
 	if (readpass(&passwd, "Please enter passphrase",
-	    dec ? NULL : "Please confirm passphrase", 1))
+	    dec ? NULL : !devtty ? NULL :
```

Comment:
Colin Percival writes ("Re: [scrypt] Provide -P option to read password from stdin (#34)"):
> > @@ -134,7 +138,8 @@ main(int argc, char *argv[])
> >  	/* Prompt for a password. */
> >  	if (readpass(&passwd, "Please enter passphrase",
> > -	    dec ? NULL : "Please confirm passphrase", 1))
> > +	    dec ? NULL : !devtty ? NULL :
> 
> I'm not sure what the point of this is... if we're not using a tty, we won't be printing a prompt anyway, I think?

I may have misunderstood the code.  Feel free to drop that aspect of
it.  Would you like me to retest without that change ?

Thanks,
Ian.